src/Media.Plugin.Android/MediaPickerActivity.cs

```diff
@@ -174,6 +174,19 @@ private void Touch()
             File.Create(GetLocalPath(this.path)).Close();
         }
 
+        private void DeleteOutputFile()
```

Comment:
Can we try/catch and output any exception to Debug.WriteLine?

Since this is new I am all about being safer.